components/service/glean/README.md

```diff
@@ -1,9 +1,9 @@
 # [Android Components](../../../README.md) > Service > Glean
 
-A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service
-(eventually replacing [service-telemetry](../telemetry/README.md)).
+A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service.
+It is a replacment for the deprecated [service-telemetry](../telemetry/README.md).
 
-Visit the [complete Glean documentation](https://mozilla.github.io/glean/). 
+Visit the [complete Glean documentation](https://mozilla.github.io/glean/).
```

Comment:
This is the right link for this audience, but do we want to say "Glean SDK documentation" here?

Reply:
Sure, will update while I'm here anyway

Reply:
```suggestion
Visit the [complete Glean SDK documentation](https://mozilla.github.io/glean/).
```